src/Ide/Plugin/Ormolu.hs

```diff
@@ -36,24 +43,24 @@ descriptor plId = (defaultPluginDescriptor plId)
 -- ---------------------------------------------------------------------
 
 provider :: FormattingProvider IO
-provider _lf ideState typ contents fp _ = do
+provider lf ideState typ contents fp _ = withIndefiniteProgress lf title Cancellable $ do
```

Comment:
Can we cancel operations in ghcide/hls? Should it be `NotCancellable` here

Reply:
ghcide does implement cancellations, so I think this should be `Cancellable` in general